codespell_lib/data/dictionary.txt

```diff
@@ -11915,6 +11922,7 @@ reguster->register
 rehersal->rehearsal
 rehersing->rehearsing
 reicarnation->reincarnation
+reification->ratification
```

Comment:
reunification should probably be a suggestion too, and perhaps more likely from what's been typed.

Reply:
Actually, it's a valid word: https://www.merriam-webster.com/dictionary/reification

Reply:
Well spotted @luzpaz

Reply:
Done